temoa-uiuc/data_files/UIUC_2050_cost.sql

```diff
@@ -0,0 +1,3078 @@
+INSERT INTO "MinActivity" VALUES('uiuc', 2028, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2029, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2030, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2031, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2032, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2033, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2034, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2035, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2036, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2037, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2038, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2039, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2040, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2041, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2042, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2043, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2044, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2045, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2046, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2047, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2048, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2049, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+INSERT INTO "MinActivity" VALUES('uiuc', 2050, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
+--
+INSERT INTO "MinActivity" VALUES('uiuc', 2021, 'GSLVCL', 91.925, 'kgal','');
```

Comment:
These values are not the most updated ones

Reply:
see: https://github.com/arfc/pride/blob/master/temoa-uiuc/data_files/11_uiuc.sql#L247